packages/renderer/src/lib/preferences/PreferencesCliToolsRendering.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+import { cliToolInfos } from '../../stores/cli-tools';
+import Markdown from '../markdown/Markdown.svelte';
+import SettingsPage from './SettingsPage.svelte';
+import EngineIcon from '../ui/EngineIcon.svelte';
+import EmptyScreen from '../ui/EmptyScreen.svelte';
+</script>
+
+<SettingsPage title="CLI Tools">
+  <div class="h-full">
+    <EmptyScreen
+      aria-label="no-resource-panel"
+      icon="{EngineIcon}"
+      title="No CLI Tool description found"
```

Comment:
should we add that it is not registered ?

```suggestion
      title="No CLI tool has been registered"
```